agent.go

```diff
@@ -138,6 +139,7 @@ type Agent struct {
 
 	interfaceFilter func(string) bool
 	ipFilter        func(net.IP) bool
+	ActiveTCP       bool
```

Comment:
As you mentioned in your PR description, we should use something like a bitmask here.
Ideally using the bit positions of the [`TCPType` type](https://github.com/pion/ice/blob/9afe3ad2d02084a6d1153d98c19308357e44f5e5/tcptype.go#L10).

E.g. 

```go
type AgentConfig struct {
    ...
    TCPTypes int
    ....
}


cfg := AgentConfig{
    TCPTypes: 1<<TCPTypeActive | 1<<TCPTypePassive
}
```

Reply:
As discussed, active candidates will be enabled by default.